### PR TITLE
Search dot cpan org to metacpan switch

### PR DESCRIPTION
**Pull request by oalders**
Just opening a PR for comments from the MetaCPAN team.  I'll request a review when it's actually ready.  I hope that's ok!

### CODE REVIEW

**Comment by oalders** [comment]
This is now ready for review.  Be harsh!

File: content/article/saying-goodbye-to-search-cpan-org.md [hunk 1]
```diff
@@ -0,0 +1,87 @@
...
+means.
+
+## Why is this change taking place?
+
+The maintainers behind [search.cpan.org](http://search.cpan.org) have decided
+that it's time to move on.  After many, many years of keeping this site up and
+running they have decided to pass on the torch (and the traffic) to
+[metacpan.org](https://metacpan.org).  The MetaCPAN team has been working very
+hard to prepare for the influx of new traffic (and new users).  On behalf of
+Perl users everywhere, the MetaCPAN team would like to thank all of the crew
```
Comment (dnmfarrell) [inline content/article/saying-goodbye-to-search-cpan-org.md]: Do you want to mention their names (if you know them)?

**Comment by dnmfarrell** [comment]
Hey @oalders 

This is a **high-quality** submission sir. Thank you! I'll merge and create a new branch with some minor edits on it.

.

File: content/article/saying-goodbye-to-search-cpan-org.md (hunk 1 above)
Comment (oalders) [reply content/article/saying-goodbye-to-search-cpan-org.md]: I did, but I'm not sure I know *all* of them.  So, I could thank a possibly incomplete list or thank them all with a generic blanket statement.  Do you have a preference?  I can follow up with them to get a proper list.  (Sorry, just saw this now).

Comment (dnmfarrell) [reply content/article/saying-goodbye-to-search-cpan-org.md]: Gotcha, as this is time sensitive let's not wait for the full list and stick with the blanket statement. Just let me know what you think about PR #111 and we'll publish

**Pull request merged**